ouroboros-consensus-cardano/cddl/node-to-client/localstatequery/consensus/getSystemStart.cddl

```diff
@@ -0,0 +1,2 @@
+query = 1
+result = time
```

Comment:
I think this is wrong. I just happened to look this up an `ToCBOR SystemStart` uses this definition of `ToCBOR UTCTime`: https://github.com/IntersectMBO/cardano-base/blob/master/cardano-binary/src/Cardano/Binary/ToCBOR.hs#L815

So something like
```cddl
time = [year, dayOfYear, timeOfDayPico]
year = integer
dayOfYear = int
timeOfDayPico = integer
```

I have no idea how `int` and `integer` are specified right now

Reply:
There seems to be a `bigint` in the CDDL prelude that would correspond to this.

Reply:
Thanks, fixed

Reply:
Saw your fixes and when using it I actually found an issue with the Rust implementation. The example `cuddle` generated from that cddl was using a negative `Int` for `dayOfYear` and the pallas decoder failed on this test (I just wrote): https://github.com/txpipe/pallas/pull/638/files#diff-5c5e18f1f6882b18dc576ed8b5c38bbe0e017339ad002dbebbf03038e150bcf9R839
with

```
thread 'miniprotocols::localstate::queries_v16::codec::tests::test_api_example_roundtrip' panicked at pallas-network/src/miniprotocols/localstate/queries_v16/codec.rs:840:25:
error decoding cbor from query message: Error { err: TypeMismatch(Tag), pos: Some(1), msg: "expected u32" }
```

Reply:
I read this comment of yours as a "success" of having a CDDL that reflects the values that actually _could_ appear in the encoding no? or is it a problem?